### ISSUE TITLE
"linking error, not in local cache" when running `npm run ci`

### ISSUE THREAD

**Issue by akauppi** [akauppi/firebase-jest-testing#5]
This is a mystery.

```
...
 FAIL  sample/test-fns/userInfo.test.js
  ● Test suite failed to run

    linking error, not in local cache
```

Also covered in https://github.com/akauppi/GroundLevel-firebase-es/issues/31

**Comment by akauppi** [comment]
Jest has solved this; once we get a new 27.0.0.next candidate, the problem should go away.

**Comment by akauppi** [comment]
27.0.0.next4 fixes this; `master` updated
